rclpy/rclpy/node.py

```diff
@@ -1031,7 +1031,7 @@ def create_publisher(
         qos_profile: Union[QoSProfile, int] = None,
         *,
         callback_group: Optional[CallbackGroup] = None,
-        event_callbacks: Optional[PublisherEventCallbacks] = None,
+        event_callbacks: Optional[PublisherEventCallbacks] = None
```

Comment:
The trailing commas that are being cause a SyntaxError reported during test_flake8.

```
36: Test command: /usr/bin/python3 "-u" "/root/ros2_ws/install/ament_cmake_test/share/ament_cmake_test/cmake/run_test.py" "/root/ros2_ws/build/rclpy/test_results/rclpy/flake8.xunit.xml" "--package-name" "rclpy" "--output-file" "/root/ros2_ws/build/rclpy/ament_flake8/flake8.txt" "--command" "/root/ros2_ws/install/ament_flake8/bin/ament_flake8" "--xunit-file" "/root/ros2_ws/build/rclpy/test_results/rclpy/flake8.xunit.xml"
36: Test timeout computed to be: 60
36: -- run_test.py: invoking following command in '/root/ros2_ws/src/ros2/rclpy/rclpy':
36:  - /root/ros2_ws/install/ament_flake8/bin/ament_flake8 --xunit-file /root/ros2_ws/build/rclpy/test_results/rclpy/flake8.xunit.xml
36:
36: ./rclpy/node.py:1035:5: E999 SyntaxError: invalid syntax
36:     ) -> Publisher:
36:     ^
36:
36: 1     E999 SyntaxError: invalid syntax
```